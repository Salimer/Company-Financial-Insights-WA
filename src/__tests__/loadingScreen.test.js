import React from 'react';
import { render } from '@testing-library/react';
import LoadingScreen from '../components/loadingScreen';

describe('LoadingScreen', () => {
  test('renders correctly', () => {
    const { container } = render(<LoadingScreen />);
    expect(container).toMatchSnapshot();
  });
});
