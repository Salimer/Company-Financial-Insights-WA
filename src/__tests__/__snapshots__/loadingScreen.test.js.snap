// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingScreen renders correctly 1`] = `
<div>
  <div
    class="sc-bgqQcB cFcgbR"
  >
    <div
      class="sc-gTRrQi byNTRp"
    />
  </div>
</div>
`;
